pyscript.core/src/core.js

```diff
@@ -213,6 +213,7 @@ define("py", {
             // resolve PyScriptElement to allow connectedCallback
             element._pyodide.resolve(pyodide);
         }
+        console.info("[pyscript/main] PyScript Ready");
```

Comment:
this is eventually `console.debug` ... so that we don't show this stuff by default but it's still visible. Please note that this would also log per every script or py-script element on the page so it should rather be flagged once outside this scope, if we really need log this.

I am also not sure why we prefix it as `[pyscript/main]` but I also don't have any strong opinion about what we shold write there, just trying to understand what `pyscript/main` means, thanks.

Reply:
This needs to be explicitly talked and decided yet. For now, I've just put a placeholder to make the test work.

Reply:
@fpliger can we please make it a `console.debug` instead? that's what I am sure MicroPython and I think Pyodide are doing too (if not, Pyodide should /cc @hoodmane)